client/public/index.html

```diff
@@ -21,15 +21,6 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
 
-    <!-- Google Tag Manager -->
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-      })(window,document,'script','dataLayer','GTM-NMPT5JP');</script>
-    <!-- End Google Tag Manager -->
```

Comment:
Err, so this should really stay because while the React app is initializing, we want anything that uses `window.dataLayer` to still work... with this gone, `window.dataLayer` will be undefined, so anything attempting to use GTM before the React app is initialized will throw an exception.  It also means that those initial data layer events that are added in this snippet (`gtm.start` etc) won't be transmitted to GTM.

Err wait, that said, I'm now noticing that this is also adding a `<script>` tag for `gtm.js` to the the page which isn't good... aghc

Reply:
Holy cow this is a completely different script tag than the one I saw in https://github.com/JustFixNYC/who-owns-what/pull/284!  It looks like the one you're removing here was originally added 10 months ago in #179, and is explicitly Whole Whale-related, while the one I fiddled with in #284 was originally added two years ago in 5e7bccb40a1b5048c928d1fd453bfc6c853ba47b.  Ahhhh why were there two GTMs on our page